ext/dom/html_document.c

```diff
@@ -545,12 +545,11 @@ static bool dom_decode_encode_fast_path(
 		const lxb_char_t *buf_ref_backup = buf_ref;
 		lxb_codepoint_t codepoint = lxb_encoding_decode_utf_8_single(&decoding_encoding_ctx->decode, &buf_ref, buf_end);
 		if (UNEXPECTED(codepoint > LXB_ENCODING_MAX_CODEPOINT)) {
-			size_t skip = buf_ref - buf_ref_backup; /* Skip invalid data, it's replaced by the UTF-8 replacement bytes */
 			if (!dom_process_parse_chunk(
 				ctx,
 				document,
 				parser,
-				buf_ref - last_output - skip,
+				buf_ref_backup - last_output,
```

Comment:
Is this unrelated to the perf optimisation commit?

Reply:
The fasr path has the same structure as this code, and I noticed the skip variable was useless. So yeah it's more like cleanup.